packages/client/hmi-client/src/components/workflow/ops/model-comparison/tera-model-comparison.vue

```diff
@@ -287,9 +287,47 @@ async function buildJupyterContext() {
 	}
 }
 
+function hasNonEmptyValue(obj) {
+	return Object.values(obj).some((value) => {
+		if (Array.isArray(value)) {
+			return value.length > 0;
+		}
+		if (typeof value === 'object' && value !== null) {
+			return Object.keys(value).length > 0;
+		}
+		return value !== null && value !== undefined && value !== '';
+	});
+}
```

Comment:
You can use [Lodash isEmtpy](https://lodash.com/docs/4.17.15#isEmpty)

```suggestion
function hasNonEmptyValue(obj) {
	return Object.values(obj).some((value) => !isEmpty(value));
}
```

Reply:
fixed